docker/config/local_dev.env

```diff
@@ -85,22 +85,17 @@ processor.command_pathname=/stackwalk/stackwalker
 
 # Set the telemetry bucket name explicitly
 destination.telemetry.bucket_name=telemetry_bucket
+telemetry.bucket_name=telemetry_bucket
 
 # webapp
 # ------
 
 ALLOWED_HOSTS=localhost,webapp
-AWS_HOST=localstack-s3
-AWS_PORT=5000
-AWS_ACCESS_KEY=foo
-AWS_SECRET_ACCESS_KEY=foo
```

Comment:
None of the `AWS*` settings are used in the webapp code. I removed them from here and the configuration files.